### PR TITLE
Use demo tileserver for devel without authToken

### PR DESCRIPTION
**Pull request by yurikgbp**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,3 +10,5 @@ Changelog rules:
 ## Unreleased
 - BC break: `coords` property requires array of arrays to support multiple points with markers.
 - Add optional `center` property to set default map center.
+- Add eslint with typescript options
```
Comment (OndraM) [inline CHANGELOG.md]:
```suggestion
- Add eslint with typescript options.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -10,3 +10,5 @@ Changelog rules:
...
+- Use demo tileserver for devel without authToken, but with max rate limit for tile requests
```
Comment (OndraM) [inline CHANGELOG.md]:
```suggestion
- Use demo tileserver for devel without authToken, but with max rate limit for tile requests.
```

File: README.md [hunk 3]
```diff
@@ -42,14 +42,13 @@ import LmcMaps from '@lmc-eu/lmc-maps';
...
-        center: [15.524, 49.766]
+        coords: [[15.524, 49.766], [15.481, 49.758]],
+        center: [15.520, 49.760]
         zoom: 7,
         style: 'lmc-default',
         marker: true,
         lang: 'cs',
-        authToken: 'YOUR_TOKEN',
-        publicUrl: 'https://tileserver.lmc.cz'
+        authToken: 'YOUR_TOKEN'
```
Comment (OndraM) [inline README.md]:
```suggestion
        // For development purposes authToken is not needed - but requests will have low rate limit.
        // authToken: 'YOUR_TOKEN',
        // Uncomment following publicUrl if you have your own authToken:
        // publicUrl: 'https://tileserver.lmc.cz'

```

File: README.md [hunk 4]
```diff
@@ -59,21 +58,27 @@ import LmcMaps from '@lmc-eu/lmc-maps';
...
 
 #### Supported languages <a name="supported-languages"></a>
 
 Currently are supported these languages:
 
 bg (Bulgarian), bs (Bosnian), cs (Czech), da (Danish), de (German), el (Greek), en (English), es (Spanish), et (Estonian), fi (Finnish), fr (French), hr (Croatian), hu (Hungarian), is (Icelandic), it (Italian), lt (Lithuanian), lv (Latvian), mk (Macedonian), nl (Dutch), pl (Polish), pt (Portuguese), ro (Romania), ru (Russian), sk (Slovak), sl (Slovene), sq (Albanian), sr (Serbian), sv (Swedish), tr (Turkish), uk (Ukrainian)
 
+#### Tileserver URL <a name="tileserver"></a>
+
+For development is set url to demo tileserver (https://tileserver.lmc.cz/demo) without authToken as default (not need to set publicUrl manually), but with rate limits for tiles requests. However, for production usage you need to set `authToken` and use production tileserver (default https://tileserver.lmc.cz).
```
Comment (OndraM) [inline README.md]:
```suggestion
Default tileserver `publicUrl` is set to demo tileserver (`https://tileserver.lmc.cz/demo`), which could be used during development. It does not need `authToken`, but is has low rate limits for tiles requests and **must not** be used on production.

 For production usage you need to set `authToken` and use production tileserver (which is eg. `https://tileserver.lmc.cz`).
```

Comment (yurikgbp) [reply README.md]: Edited.

File: README.md (hunk 3 above)
Comment (yurikgbp) [reply README.md]: Edited.

**Pull request merged**